cmd/daemon/service.go

```diff
@@ -0,0 +1,72 @@
+//go:build windows
+// +build windows
```

Comment:
The functions in this file have been taken from the windows/svc source code right? Should we add a reference for that in the file doc?

Reply:
+1

Reply:
Not exactly, I did look at what was being done there for some help with the waitgroups, but the code is mostly the same as the example given in the svc library https://github.com/golang/sys/blob/master/windows/svc/example/service.go